test/randomActionAgent.py

```diff
@@ -0,0 +1,70 @@
+import random
+import numpy as np
+from pathlib import Path
+
+from animalai import AnimalAIEnvironment, arenas
+from animalai.actions import AAIActions
+from animalai.executable import find_executable
+from animalai.agents import RandomActionAgent
+
+
+def main():
+    """
+    Manual test to see if random action agent works.
+    """
+
+    env = AnimalAIEnvironment(
+        file_name=str(find_executable(Path("test/executable/"))),
+        arenas_configurations=arenas.GOOD_GOAL_RANDOM_POS,
+        base_port=5005 + random.randint(0, 1000),
+        useRayCasts=True,
+        rayMaxDegrees=30,
+        raysPerSide=3,
+        play=False,
+        inference=True,
+    )
+
+    randomAgent = RandomActionAgent(step_length_distribution=lambda: np.random.normal(
+        5, 1))  # a Rayleigh walker (sampling from normal distribution)
+
+    # by default should be AnimalAI?team=0
+    behavior = list(env.behavior_specs.keys())[0]
+
+    # define actions
+    actions = AAIActions()
+
+    # Run two episodes
+    for _episode in range(2):
```

Comment:
do the episodes start at index 0? so, arena 0, arena 1, ... , n arenas.

Did i misunderstand this loop?

Reply:
It plays the config twice over. You can learn more about for loops in python [here](https://pythonexamples.org/python-for-i-in-range/).